src/app.js

```diff
@@ -28,7 +28,9 @@ app.use(session({
   cookie: {
     httpOnly: true,
     sameSite: 'strict',
+    maxAge: Number(process.env.SESSION_TIMEOUT),
   },
+  rolling: true,
```

Comment:
What does `rolling` do?

Reply:
Every call to the API resets the timeout on the cookie. https://www.npmjs.com/package/express-session#rolling